src/js/builtins/Module.ts

```diff
@@ -3,14 +3,16 @@ export function main() {
   return $requireMap.$get(Bun.main);
 }
 
-$visibility = "Private";
+$visibility = "PrivateRecursive";
+$alwaysInline;
```

Comment:
i wonder if this actually does anything. this function is almost always `.bind()`ed to the module record.

Reply:
Ya, this PR was mainly a "see something, say something" moment. I think knowing we have the ability to specify `$awaysInline` is powerful. Not likely the place for it though you are right.